core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -471,6 +471,16 @@ package object config {
         "cache block replication should be positive.")
       .createWithDefaultString("30s")
 
+  private[spark] val STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH =
```

Comment:
Because we only register fallback storage when SparkContext initialization, I think it doesn't make sense to change this config after creating SparkContext, so this sounds like a static SQL config?

Reply:
This is `CORE` module which can work without SQL module. So, we cannot put it to Static SQL conf, @viirya .

Reply:
Oh I see. Seems we don't have static config for core.

Reply:
Could you add a simple note to the doc? E.g., "Fallback storage is registered during SparkContext initialization, so this must be enabled before creating SparkContext."

Reply:
Ur, but it will be the same for all decommission conf like `spark.storage.decommission.enabled` and the other general conf like `spark.driver.host` and `spark.driver.port`.

Reply:
Oh okay.